scripts/create_pelorus_operator

```diff
@@ -115,7 +115,10 @@ pushd "${destination_dir}" || exit 2
     # Correct operator version, to be latest +1
     OPERATOR_VERSION=$(curl -H "Authorization: Bearer XYZ" -X GET "https://quay.io/api/v1/repository/${OPERATOR_ORG_NAME}/${OPERATOR_PROJECT_NAME}/tag/" | jq .tags[].name | head -1 | sed -e 's|\"||g')
     echo "INFO: Current operator version: ${OPERATOR_VERSION}"
-    NEW_OPERATOR_VERSION=$(echo ${OPERATOR_VERSION} | awk -F. -v OFS=. '{$NF += 1 ; print}')
+    NEW_OPERATOR_VERSION=$(echo "${OPERATOR_VERSION}" | awk -F. -v OFS=. '{$NF += 1 ; print}')
+    if [ "$NEW_OPERATOR_VERSION" -eq "1" ]; then
```

Comment:
did not understand this. The version will not always be in the `major.minor.patch` style?

Reply:
When this line gives error, meaning the operator was not found in this org, then the following line will always result to "1":
```
OPERATOR_VERSION=$(curl -H "Authorization: Bearer XYZ" -X GET "https://quay.io/api/v1/repository/${OPERATOR_ORG_NAME}/${OPERATOR_PROJECT_NAME}/tag/" | jq .tags[].name | head -1 | sed -e 's|\"||g')

# This will result to "1"
NEW_OPERATOR_VERSION=$(echo "${OPERATOR_VERSION}" | awk -F. -v OFS=. '{$NF += 1 ; print}')
```

In the above case we want to have version in the major.minor.patch, so we set the first one as `0.0.1`